pkg/apis/velero/v1/restore_types.go

```diff
@@ -123,6 +123,19 @@ type RestoreSpec struct {
 	// +optional
 	// +nullable
 	ResourceModifier *v1.TypedLocalObjectReference `json:"resourceModifier,omitempty"`
+
+	// UploaderConfig specifies the configuration for the restore.
+	// +optional
+	// +nullable
+	UploaderConfigForRestore *UploaderConfigForRestore `json:"uploaderConfig,omitempty"`
```

Comment:
Trivial: for simplicity and consistency with json key and comment, the name of the Field in the struct should better be `UploaderConfig`, there won't be confusion, b/c normally the code would look like 
```
restore.Spec.UploaderConfig .....
```

Reply:
updated